### ISSUE TITLE
Link Between Slides?

### ISSUE THREAD

**Issue by jasonwhat** [alleyinteractive/revealjs-wp-theme#2]
Is it possible to link between slides? There are some fun things one could do with this theme, but it would need more settings to go beyond one-off slide experience like the ability to have multiple slideshows, or at least link between slides to give that effect.

Overall, this is really impressive. Nice work.

**Comment by mboynes** [comment]
That's a great idea. You can currently link between slides [as detailed in the reveal.js documentation](https://github.com/hakimel/reveal.js/#internal-links), but it would be nice if each slide had an "ID" field that could be used for the second approach.

**Comment by jasonwhat** [comment]
In the current setup, would I link based on slide numbers and I simply need to know the slide number?
    `<a href="#/2/2">Link</a>`

The above would link to the second vertical slide of slide 2?

**Comment by mboynes** [comment]
Yes, I believe that's exactly right. Personally, I would just navigate to the slide I wanted to link to, and copy it from the address bar.
